app/Http/Controllers/Api/ProjectController.php

```diff
@@ -44,88 +59,152 @@ public function add_project(Request $request){
         try{
             $result = Project::create([
                 'user_id'=> $request->user()->id,
-                'project_address_id' => $request->project_address_id,
+                'builder_category_id' => null,
```

Comment:
Why are you making the builder_category_id as a required field in validation and then putting null here?

Reply:
resolved